globus_sdk/transfer/data.py

```diff
@@ -25,7 +25,7 @@ class TransferData(dict):
     ``"exists"``, ``"size"``, ``"mtime"``, or ``"checksum"`` if you want
     greater clarity in client code.
 
-    Includes fetching the submission ID as part of document generation. The
+    If `submission_id` isn't passed, one will be fetched automatically. The
```

Comment:
I hate to nitpick this, but ReST requires double-ticks for this.
Applies to `DeleteData` below as well.

Let me know when that's fixed and I'll merge.